### ISSUE TITLE
API Key

### ISSUE THREAD

**Issue by vikingosegundo** [car2go/openAPI#12]
Where can I obtain an api key? Thanks

**Comment by DSIW** [comment]
You have to write an email to Car2Go, see [Registration information](https://github.com/car2go/openAPI/wiki/Access-protected-Functions-via-OAuth-1.0#registration-as-consumer).

**Comment by dchasepdx** [comment]
Did you get your api key? About how long does it usually take?

**Comment by DSIW** [comment]
Yes, I got my credentials in 4 days.

**Comment by knaufk** [comment]
Do I also need an API Key for the public APIs?

**Comment by DSIW** [comment]
Yes, I think everyone needs an API key. I tried a public endpoint via `GET https://www.car2go.com/api/v2.1/locations` and it responds with `Error 401: Invalid consumerkey`.

**Comment by ajespin** [comment]
I didn´t have request from openapi@car2go.  i have like 2 weeks waiting
what can i do?
thank you !!

**Comment by tonyjayking** [comment]
I wait for one week... No, any response yet

**Comment by skorpach** [comment]
I requested a key last week, they responded with a form letter stating that they are not granting keys to new partners.

**Comment by patoroco** [comment]
Hi,

sorry but I don't see _Registration Information_ in wiki-page.

Are access keys available?

**Comment by skorpach** [comment]
Here's a link to the Terms of Use:  https://www.car2go.com/api/tou.htm
2.b is the section on applying for a licence. In short send an email to openapi@car2go.com

Car2Go does not seem to be issuing new keys at this time, however, so expect a rejection letter in a few weeks.

**Comment by canales** [comment]
Hi, our certificate for the API just expired and we have been cutoff from the API (which we use to show vehicles in our screens). I already sent an email to openapi@car2go.com but haven't got any response. If you could look into this, I'd appreciate it.
